### PR TITLE
[sui-dashboard] First draft

### PR DESCRIPTION
**Pull request by carlosvillu**
- [x] README

### CODE REVIEW

File: packages/sui-dashboard/bin/sui-dashboard-components.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  'progress false',
+  'loglevel error',
+  'ignore-scripts',
+  'save-exact',
+  'production',
+  // "fetch-retry-factor 0",
+  // "fetch-retries 0",
+  'json'
+]
+const repositories = [
```
Comment (carlosvillu) [inline packages/sui-dashboard/bin/sui-dashboard-components.js]: Did I forget a repo ?!

File: packages/sui-dashboard/bin/sui-dashboard-components.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  'no-shrinkwrap',
+  'no-optional',
+  'no-bin-links',
+  'prefer-offline',
+  'progress false',
+  'loglevel error',
+  'ignore-scripts',
+  'save-exact',
+  'production',
+  // "fetch-retry-factor 0",
```
Comment (carlesnunez) [inline packages/sui-dashboard/bin/sui-dashboard-components.js]: should we delete this?

File: packages/sui-dashboard/src/index.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      percentage:
+        Math.ceil(
+          (partialStats.suiStats.totalSUIComponents * 100) /
+            partialStats.suiStats.maxPossible
+        ) + '%'
+    }
+  }
+}
+
+// Dead code, but I dont want to remove
```
Comment (carlesnunez) [inline packages/sui-dashboard/src/index.js]: Why?

File: packages/sui-dashboard/bin/sui-dashboard-components.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+/* eslint no-console:0 */
+
+const os = require('os')
+const path = require('path')
+const program = require('commander')
+const {parallelSpawn} = require('@s-ui/helpers/cli')
+const {stats, excel} = require('../src')
```
Comment (jcger) [inline packages/sui-dashboard/bin/sui-dashboard-components.js]: you could remove excel, looks like it isn't used

File: packages/sui-dashboard/src/index.js (hunk 3 above)
Comment (carlosvillu) [reply packages/sui-dashboard/src/index.js]: I really like it !!! 😅

File: packages/sui-dashboard/bin/sui-dashboard-components.js (hunk 2 above)
Comment (carlosvillu) [reply packages/sui-dashboard/bin/sui-dashboard-components.js]: sure!

File: packages/sui-dashboard/src/index.js (hunk 3 above)
Comment (carlesnunez) [reply packages/sui-dashboard/src/index.js]: 🤣  So let's keep that deadly code here 👍

**Pull request merged**